ChainSafe/web3.unity
# MetaMask error pop up.

MetaMask logout fixed with task completion source. Closes #1017 

To Test:

Boot up SDK and build to Webgl,
Reject the login request with MetaMask.

Expected Results:

The error pop up will be presented on screen like so:

![image](https://github.com/ChainSafe/web3.unity/assets/57473220/b31183a1-80f1-4e99-ac8e-7a618d2ad9c8)
